CRM/EventsExtras/Form/Settings.php

```diff
@@ -66,6 +66,13 @@ public function buildQuickForm() {
   public function postProcess() {
     $configFields = SettingsManager::getConfigFields();
     $submittedValues = $this->exportValues();
+    $paymentProcessors = $submittedValues[SettingsManager::SETTING_FIELDS['PAYMENT_PROCESSOR_SELECTION_DEFAULT']];
+    //flip selected default paymentProcessors before saving to setting
+    $flipedPaymentProcessor = [];
```

Comment:
Can we use `$flipedPaymentProcessor = array_keys($paymentProcessors)` ?

Reply:
This is outdate.